client/my-sites/plans/main.jsx

```diff
@@ -166,7 +167,9 @@ class Plans extends Component {
 	};
 
 	componentDidMount() {
+		const { selectedSite } = this.props;
 		this.redirectIfInvalidPlanInterval();
+		this.props.fetchSitePlans( selectedSite.ID );
```

Comment:
This may need to check selectedSite.ID is something truthy before fetching - ignore if I'm wrong, I haven't looked at react/redux like this in a while.

Reply:
Added checks for it.